auto-generated/cluster/stone-prd-rh01/managed/rh-managed-cnv-fbc-tenant/appstudio.redhat.com_v1alpha1_releaseplanadmission_fbc-staged-index-release-v4-15-plus.yaml

```diff
@@ -9,13 +9,18 @@ spec:
   - cnv-fbc-v4-16
   data:
     fbc:
+      allowedPackages:
+      - kubevirt-hyperconverged
       binaryImage: registry.redhat.io/openshift4/ose-operator-registry-rhel9:v4.15
       buildTimeoutSeconds: 1500
       fbcPublishingCredentials: staged-index-fbc-publishing-credentials
       fromIndex: registry-proxy.engineering.redhat.com/rh-osbs/iib-pub-pending:v4.13
       requestUpdateTimeout: 1500
       stagedIndex: true
       targetIndex: ""
+    releaseNotes:
```

Comment:
I don't know why we are adding releaseNotes to fbc RPAs, but others have it too so :man_shrugging:

Reply:
Yes, I am also only adding to match with others, but it is definitely not required.